examples/samples/topicFilters.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  Namespace,
+  SubscriptionClient,
+  TopicClient,
+  SendableMessageInfo,
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage
+} from "../../lib";
+import * as dotenv from "dotenv";
+import { CorrelationFilter } from "../../lib/core/managementClient";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const subscriptionAllMessages = process.env.SUBSCRIPTION_ALL_MESSAGES || "";
+const subscriptionColorBlueSize10Orders = process.env.SUBSCRIPTION_COLOR_BLUE_ORDER_10 || "";
+const subscriptionColorRed = process.env.SUBSCRIPTION_CORRELATION_COLOR_RED || "";
+const subscriptionHighPriorityOrders = process.env.SUBSCRIPTION_HIGH_PRIORITY || "";
+
+console.log("str: ", str);
+console.log("path: ", topic);
+console.log("Subscription 1: ", subscriptionAllMessages);
```

Comment:
I think we should use the actual name of the subcription (ie. instead of 1, use something like SUBSCRIPTION_ALL_MESSAGES)

Reply:
Okay!